nikolai4D/frontEndFramework
# Uc 96 create modal for user in navbar

UC-96-create-modal-for-user-in-navbar
## Description & motivation
created the modal for usermodal

created the modal for usermodal


## How to test
needs UC-96-create-modal-for-user-in-navbar from UrbanCloud-alt1
run app
run build
go to search view and modal should be there

once the steps above are done, go into the search view.


## New tickets to be added





---
